### PR TITLE
Doc: Copy project

### PR DESCRIPTION
**Pull request by ZJvandeWeg**

### CODE REVIEW

File: frontend/src/pages/project/Settings/Danger.vue [hunk 1]
```diff
@@ -46,11 +46,11 @@
             </div>
         </div>
 
-        <FormHeading>Export Project</FormHeading>
+        <FormHeading>Copy Project</FormHeading>
         <!-- Hiding for now (0.5) -->
```
Comment (ZJvandeWeg) [inline frontend/src/pages/project/Settings/Danger.vue:50]: Not sure what this is, or what needs to happen with it?

Comment (hardillb) [reply frontend/src/pages/project/Settings/Danger.vue:50]: This is to move a project out of FlowForge (exports flows and creds). Given we now have projects not sure we need this at the moment.

File: frontend/src/pages/project/Settings/Danger.vue [hunk 2]
```diff
@@ -46,11 +46,11 @@
...
         <!-- <div>
             <div>
-                <ff-button kind="secondary" @click="showExportProjectDialog()">Export Project</ff-button>
+                <ff-button kind="secondary" @click="showExportProjectDialog()">Copy Project</ff-button>
```
Comment (hardillb) [inline frontend/src/pages/project/Settings/Danger.vue:53]:
```suggestion
                <ff-button kind="secondary" @click="showExportProjectDialog()">Export Project</ff-button>
```

Reverting this as Export is right for this action if it was re-enabled.

File: frontend/src/pages/project/Settings/dialogs/ExportProjectDialog.vue [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <ff-dialog ref="dialog" header="Export Project" confirm-label="Export Project" @confirm="confirm()">
+    <ff-dialog ref="dialog" header="Copy Project" confirm-label="Copy Project" @confirm="confirm()">
```
Comment (hardillb) [inline frontend/src/pages/project/Settings/dialogs/ExportProjectDialog.vue:2]:
```suggestion
    <ff-dialog ref="dialog" header="Export Project" confirm-label="Export Project" @confirm="confirm()">
```

Again reverting as Export is the correct way to describe this (if it gets re-enabled). We should schedule a review of this feature for after Thursday.

**Review by hardillb — APPROVED** [review]

**Pull request merged**